data/tlg0007/tlg081/__cts__ .xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:greekLit:tlg0007"
+    projid="greekLit:tlg081" urn="urn:cts:greekLit:tlg0007.tlg083" xml:lang="grc">
+    <ti:title xml:lang="lat">Regum et imperatorum apophthegmata</ti:title>
+
+    <ti:translation urn="urn:cts:greekLit:tlg0007.tlg081.perseus-eng3"
+        workUrn="urn:cts:greekLit:tlg0007.tlg081" xml:lang="eng">
+        <ti:label xml:lang="eng">SAYINGS OF KINGS AND COMMANDERS</ti:label>
+        <ti:description xml:lang="mul">Plutarch. Moralia, Vol. III. Babbitt, Frank Cole, translator.
+            Cambridge, MA: Harvard University Press. London: William Heinemann Ltd. 1931
+            (printing).</ti:description>
+    </ti:translation>
+
+    <ti:translation urn="urn:cts:greekLit:tlg0007.tlg081.perseus-eng4"
+        workUrn="urn:cts:greekLit:tlg0007.tlg081" xml:lang="eng">
+        <ti:label xml:lang="eng">The apophthegms or remarkable sayings of kings and great
+            commanders.</ti:label>
+        <ti:description xml:lang="eng">Plutarch. Plutarch's Morals, Vol. I. Goodwin, William W.,
+            editor; Hinton, E., translator. Boston: Little, Brown, and Company. Cambridge.
```

Comment:
The first name is Edward via http://catalog.perseus.org/catalog/urn:cts:greekLit:tlg0007.tlg081.perseus-eng2